spring-cloud-starter-alibaba/spring-cloud-starter-alibaba-nacos-config/src/main/java/com/alibaba/cloud/nacos/endpoint/NacosConfigEndpointAutoConfiguration.java

```diff
@@ -52,7 +53,7 @@ public NacosConfigEndpoint nacosConfigEndpoint() {
 
 	@Bean
 	@ConditionalOnMissingBean
-	@ConditionalOnProperty(name = "spring.cloud.nacos.config.health.enabled", matchIfMissing = true)
+	@ConditionalOnEnabledHealthIndicator("nacosConfig")
```

Comment:
`@ConditionalOnEnabledHealthIndicator("nacos-config")` is better.

Reply:
> `@ConditionalOnEnabledHealthIndicator("nacos-config")` is better.

fixed